index.html

```diff
@@ -4,42 +4,175 @@
+
+    <div class="fifth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
     </div>
 
+    <div class="sixth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
 </section>
 
+<section class="item-3">
+    <div class="about">
+
+        <h2 class="aboutMe">About me</h2>
+        <p class="decription">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="seeProject">Get My Resume</a>
+
+        <img class="bottomline" src="assets/buttomline.png" alt="border line">
+
+        <div class="language">
+            <div class="triangle">
+                <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle">
+                <h2 class="lang">Language</h2>
+            </div>
+            <ul class="langElement">
+                <li class="javascript">Javascript</li>
+                <li class="ruby">Ruby</li>
+                <li class="html">Html</li>
+                <li class="css">Css</li>
+            </ul>
+
+        </div>
+
+        <div class="framework">
+            <div class="dot">
+                <img class="rect" src="assets/square.png" alt="square">
+                <h2 class="lang">Framework</h2>
+            </div>
+            <ul class="frameElement">
+             <li class="boot">Bootstrap</li>
+            <li class="rubyRail">Ruby on Rails</li>
+            <li class="rspes">RSPes</li>
+            <li class="capybara">CapyBara</li>
+            <li class="sele">Selenium</li>
+            </ul>            
+
+        </div>
+
+        <div class="skill">
+            <div class="circle">
+                <img class="rect" src="assets/circle.png" alt="circle">
+                <h2 class="lang">Skill</h2>
+            </div>
+            <ul class="skillElement">
+             <li class="codekit">Codekit</li>
+            <li class="github">Github</li>
+            <li class="codepen">Codepen</li>
+            <li class="gitlab">Gitlab</li>
+            <li class="terminal">Terminal</li>
+            </ul>         
+
```

Comment:
- [ ] Please correct your indentations and remove unnecessary blank lines. This makes for better code readability and organization.